src/TestUtils/src/DeviceTests/AssertionExtensions.cs

```diff
@@ -236,5 +236,26 @@ public static bool IsLoadedOnPlatform(this IElement element)
 
 			return pvh.PlatformView?.IsLoaded() == true;
 		}
+
+		public static async Task AssertEventually(this Func<bool> assertion, int timeout = 1000, int interval = 100, string message = "Assertion timed out")
```

Comment:
Can this call into AssertionExtentions.Wait?  

It's always kind of annoyed me that "Wait" doesn't throw an XunitException. 

https://github.com/dotnet/maui/blob/03347415fe7d3a67b69072fd5513d94f25d6222f/src/TestUtils/src/DeviceTests/AssertionExtensions.cs#L17

Just curious if there's a way to merge this a bit with `Wait`

Reply:
I'm not sure I understand what you're asking for. 
I explicitly didn't use `Wait()` because it randomizes the intervals and doesn't throw an `XunitException`. 

> Just curious if there's a way to merge this a bit with Wait

I disagree with the implementation of `Wait` and would be happy to replace it, but that should be in another PR explicitly for that purpose. You should open an issue :)

Reply:
https://github.com/dotnet/maui/issues/19376